Form-Controls/index.html

```diff
@@ -13,15 +13,66 @@
     <h1>Product Pick</h1>
   </header>
   <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
+    <form action="result.html" method="get">
+        <div class="form-element">
+          <label for="name">Name</label>
+          <input type="text" name="name" id="name" placeholder="username" required>
```

Comment:
If I added a name that was just a 1, what would happen?

Reply:
The form will accept it as a valid name, I might need to consider adding additional validations.